### PR TITLE
PWN-5773 - Minor fixes for transfer transactions details

### PR DESCRIPTION
**Pull request by 1sakov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-5773

## Description of Work

[ what work was done in this ticket ]

## Screenshots (Optional)

Screenshot #1 | Screenshot #2 | Screenshot #3
:------------:|:-------------:|:------------:
[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]

### CODE REVIEW

File: app/src/main/res/layout/dialog_history_transaction_details.xml [hunk 1]
```diff
@@ -143,101 +146,91 @@
...
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginHorizontal="16dp"
                 android:layout_marginTop="24dp"
                 android:orientation="horizontal">
 
                 <TextView
-                    android:id="@+id/destinationTitleTextView"
-                    android:layout_width="0dp"
+                    android:id="@+id/feesTitleTextView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_history_transaction_details.xml:161]: textViewFeesTitle

File: app/src/main/res/layout/dialog_history_transaction_details.xml [hunk 2]
```diff
@@ -143,101 +146,91 @@
...
                     android:drawablePadding="8dp"
-                    android:text="@string/transaction_details_to"
+                    android:text="@string/transaction_details_fee"
                     android:textAppearance="@style/UiKit.TextAppearance.SemiBold.Text2"
                     android:textColor="@color/text_night" />
 
                 <TextView
-                    android:id="@+id/destinationAddressTextView"
-                    android:layout_width="0dp"
+                    android:id="@+id/feesTextView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_history_transaction_details.xml:170]: textViewFeesValue

File: app/src/main/res/layout/dialog_history_transaction_details.xml [hunk 3]
```diff
@@ -143,101 +146,91 @@
...
-                    android:layout_weight="1"
                     android:drawablePadding="6dp"
-                    android:ellipsize="middle"
+                    android:ellipsize="end"
                     android:gravity="end"
                     android:maxLines="1"
                     android:textAppearance="@style/UiKit.TextAppearance.Regular.Text2"
-                    android:textColor="@color/text_mountain"
-                    tools:text="bc1q830ddmnzc2539vmudxz44amm2x84j5yjvc209nrlw5c7s805nusq7k4tyf" />
+                    android:textColor="@color/textIconActive"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_history_transaction_details.xml:179]: shouldn't we use the colors from UiKit?

File: app/src/main/res/layout/dialog_history_transaction_details.xml [hunk 4]
```diff
@@ -143,101 +146,91 @@
...
                 android:layout_marginTop="24dp"
-                android:orientation="horizontal"
-                android:visibility="gone"
-                tools:visibility="visible">
+                android:orientation="horizontal">
 
                 <TextView
-                    android:id="@+id/textViewUsernameTitle"
-                    android:layout_width="0dp"
+                    android:id="@+id/statusTitleTextView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_history_transaction_details.xml:192]: textViewStatusTitle

File: app/src/main/res/layout/dialog_history_transaction_details.xml [hunk 5]
```diff
@@ -143,101 +146,91 @@
...
                     android:drawablePadding="8dp"
-                    android:text="@string/transaction_details_username"
+                    android:text="@string/transaction_details_status"
                     android:textAppearance="@style/UiKit.TextAppearance.SemiBold.Text2"
                     android:textColor="@color/text_night" />
 
                 <TextView
-                    android:id="@+id/textViewUsernameValue"
-                    android:layout_width="0dp"
+                    android:id="@+id/statusTextView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_history_transaction_details.xml:201]: textViewStatusValue

File: app/src/main/res/layout/dialog_history_transaction_details.xml [hunk 6]
```diff
@@ -143,101 +146,91 @@
...
-                android:id="@+id/containerActor"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginHorizontal="16dp"
                 android:layout_marginTop="24dp"
                 android:orientation="horizontal">
 
                 <TextView
-                    android:id="@+id/actorAddressTitleTextView"
+                    android:id="@+id/transactionIdTitleTextView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_history_transaction_details.xml:223]: same wrong naming

**Review by kabramovp2p — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**